website/source/docs/secrets/transit/index.html.md

```diff
@@ -374,6 +381,51 @@ only encrypt or decrypt using the named keys they need access to.
   </dd>
 </dl>
 
+### /transit/export/encryption-key/<name>(/<version>)
+### /transit/export/signing-key/<name>(/<version>)
+### /transit/export/hmac-key/<name>(/<version>)
+#### GET
+
+<dl class="api">
+  <dt>Description</dt>
+  <dd>
+    Returns the named key. The `keys` object shows the value of the key for
+    each version. If `version` is specified, the specific version will be
+    returned. If `latest` is provided as the version, the current key will be
+    provided. Depending on the type of key, different information may be
+    returned. The key must be exportable to support this operation and the
```

Comment:
There is the following line in the documentation that say it can only be set at key creation.  This is handled the same way as the `derived` and `convergent_encryption` parameters.

> The values set here cannot be changed after key creation.